ajax/libs/noti.js/package.json

```diff
@@ -16,6 +16,6 @@
   "license": "GPL-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/j-l-n/noti.js.git"
+    "url": "https://github.com/needim/noty.git"
```

Comment:
@maruilian11 hi,  I based this on my search and inspecting the repo.  The user did not respond to my email but the description and functionality are the same as far as I can tell.